include/swift/AST/ModuleDependencies.h

```diff
@@ -718,9 +781,8 @@ class ModuleDependencyInfo {
   ArrayRef<ModuleDependencyID> getSwiftOverlayDependencies() const {
     return storage->swiftOverlayDependencies;
   }
-
-  void
-  setCrossImportOverlayDependencies(const ArrayRef<ModuleDependencyID> dependencyIDs) {
+  void setCrossImportOverlayDependencies(
```

Comment:
The `clang-format` makes this patch much harder to review. Can you split out the format change on the non-functional change into a separate commit for review?

Reply:
Split out in a separate commit. 👍🏼